### ISSUE TITLE
New update ?

### ISSUE THREAD

**Issue by MohammedGeorge** [UnamSanctam/SilentCryptoMiner#2380]
It seems like there hasn't been a new update for a while. Are you considering adding something new? Also, thank you for everything

**Comment by UnamSanctam** [comment]
Yes I'm always working on the next version, though since the latest version has been pretty stable (at least from what I've been told) then I haven't prioritized releasing a new update, but I'm planning on releasing the new version pretty soon I think. The next version focuses mostly on more antivirus avoidance though there are some things like "extra data" reporting to the web panel (the amount of RAM, the amount of CPU cores and things like that).

**Comment by dlsofree** [comment]
> Yes I'm always working on the next version, though since the latest version has been pretty stable (at least from what I've been told) then I haven't prioritized releasing a new update, but I'm planning on releasing the new version pretty soon I think. The next version focuses mostly on more antivirus avoidance though there are some things like "extra data" reporting to the web panel (the amount of RAM, the amount of CPU cores and things like that).

Hope the new version will be released soon, thank you

**Comment by MohammedGeorge** [comment]
> Yes I'm always working on the next version, though since the latest version has been pretty stable (at least from what I've been told) then I haven't prioritized releasing a new update, but I'm planning on releasing the new version pretty soon I think. The next version focuses mostly on more antivirus avoidance though there are some things like "extra data" reporting to the web panel (the amount of RAM, the amount of CPU cores and things like that).

Thanks for all unam i will tip u something soon thank you!

**Comment by svox07** [comment]
will the new version of Proxy be built-in?
